src/main/service/listManipulation/CrimeListsService.ts

```diff
@@ -91,18 +92,16 @@ export class CrimeListsService {
         let linkedCases = '';
         let listingNotes = '';
 
-        if (hearing?.case) {
-            hearing.case.forEach(cases => {
-                linkedCases = '';
-                if (cases?.caseLinked) {
-                    cases.caseLinked.forEach(caseLinked => {
-                        linkedCases += caseLinked.caseId.trim();
-                        linkedCases += helperService.stringDelimiter(linkedCases?.length, ',');
-                    });
-                }
-                cases['linkedCases'] = linkedCases?.replace(/,\s*$/, '').trim();
-            });
-        }
+        hearing?.case.forEach(cases => {
```

Comment:
I've checked all the lists that call this, and case is mandatory so I've kept that as it is.

I have removed the 'hearing' optional though, as it will always be present when calling this